.github/workflows/publish-image.yml

```diff
@@ -13,6 +13,10 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v4
+      - name: Set up uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          version: 'latest'
```

Comment:
Do we need to set up uv to publish the image? Is this for later in case we want to run tests?

Reply:
Good question! I think this is a vestige from picking up an old branch and standing it up again.